src/main/groovy/com/bmuschko/gradle/docker/DockerJavaApplicationPlugin.groovy

```diff
@@ -98,42 +117,59 @@ class DockerJavaApplicationPlugin implements Plugin<Project> {
             description = "Copies the distribution resources to a temporary directory for image creation."
             dependsOn installTask
             from { installTask.destinationDir.parentFile }
-            into { createDockerfileTask.destFile.parentFile }
+            into { createDockerfileTask.destFile.get().asFile.parentFile }
```

Comment:
Does `into` support taking a provide?
If so, you can change this to:
```groovy
into = createDockerfileTask.destFile.map { it.asFile.parentFile }
```

Reply:
The `into` methods in `AbstractCopyTask` do not expose a `Provider` variant yet. Otherwise I'd agree with the change.